### PR TITLE
feat: add auto reload

### PR DESCRIPTION
**Pull request by mvarlic**
* reload 'Webpay Plus' and 'Oneclick' every 5 minutes

### CODE REVIEW

File: src/components/pageRefresh/PageRefresh.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+"use client";
+
+import { useEffect } from "react";
+
+
+export const PageRefresh = () => {
+
+  useEffect(() => {
+    const reload = async () => {
+      console.log('refresh token!!!');
```
Comment (mastudillot) [inline src/components/pageRefresh/PageRefresh.tsx:10]: remove console log

File: src/components/pageRefresh/PageRefresh.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+"use client";
+
+import { useEffect } from "react";
+
+
+export const PageRefresh = () => {
+
+  useEffect(() => {
+    const reload = async () => {
```
Comment (mastudillot) [inline src/components/pageRefresh/PageRefresh.tsx:9]: the async function is unnecessary, location.reload() is not a promise

File: src/components/pageRefresh/PageRefresh.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      console.log('refresh token!!!');
+      window.location.reload();
+    };
+    const interval = setInterval(reload, 5*60*1000);
```
Comment (mastudillot) [inline src/components/pageRefresh/PageRefresh.tsx:13]: use constant for timeout value

**Review by mastudillot — COMMENTED** [review]
check my comments

**Review by victormendoza96 — APPROVED** [review]

**Review by mastudillot — APPROVED** [review]
LGTM

**Pull request merged**